package.json

```diff
@@ -245,13 +245,17 @@
     "start-mssql": "bash dev/mssql/2019/start.sh",
     "start-db2": "bash dev/db2/11.5/start.sh",
     "start-oracle": "bash dev/oracle/21-slim/start.sh",
```

Comment:
Maybe we should remove this so it's clearer which version we are using. Also holds for the stop script

Reply:
I left the start-oracle command as is because the term would-be uniform across all the dialects. Also, a user can call the `start-oracle-<DB version>` to set up a version-specific DB.

Reply:
Yeah, on the `main` branch we have changed that already and might do so as well for `v6`. Check out https://github.com/sequelize/sequelize/pull/14737
So to be ahead of that change I would prefer to only have the Oracle DB version specific scripts defined

Reply:
Okay thanks, should I rename these to start-oracle-oldest, start-oracle-latest, etc or start-oracle-<DB version> looks good!

Reply:
If you could update to -oldest/-latest that would be great, thanks

Reply:
Sure, I would update it.

Reply:
Updated the package.json. Thanks.